freqtrade/optimize/backtesting.py

```diff
@@ -206,6 +206,11 @@ def load_bt_data(self) -> Tuple[Dict[str, DataFrame], TimeRange]:
         """
         self.progress.init_step(BacktestState.DATALOAD, 1)
 
+        if self.config.get('freqai') is not None:
```

Comment:
is freqai assumed to be active/enabled the moment a freqai key is found?
there's no `freqai.enabled` - like we have it with telegram/rpc methods - so it can easily be disabled ? 

Not saying that's necessarily a problem - more a question if you will.

Reply:
At the moment, yes, it is assumed enabled. For consistency, it makes sense to add an `enabled` parameter similar to the other config methods.

Reply:
i'm ... undecided on this.
on one hand, it would be aligned - on the other hand - it's near impossible to simply "disable" freqAI while continuing to use the same strategy.

This reminds me of another thing - "how" does `--strategy-list` work when used to compare a freqAI and non-freqAI strategy?

Reply:
Yes, but - the point is not necessarily to use the same strategy but to use the same config for a FreqAI strat and a non FreqAI strat. 

RE: `--strategy-list`
That is a scary question, mostly because my head isn't into that side of the code but I suspect there are data loading conflicts if FreqAI is not the first strategy on the list (I've never tested it, for the record). I have half a mind to simply disallow its use if `freqai` is enabled.